_data/changelogs/component-tooltip.yml

```diff
@@ -5,6 +5,13 @@ type: component
 changelogURL:
 
 items:
+  - date: NNNN-NN-NN
+    summary: "Restored the `opacity: 0` style rule to the tooltip's initial state."
+    summaryAdditional:
```

Comment:
I like this line from your PR summary as additional context:
> This prevents the component from flickering if its position needs to be recalculated.

I think it communicates the _why_ nicely

Reply:
I like that. Added in [d6c4625](https://github.com/uswds/uswds-site/pull/2265/commits/d6c46255d1fcf21e320b22970a0fb6782e4cf3ba).